### PR TITLE
Update Partners page and Partners section on Homepage

### PR DESCRIPTION
**Pull request by ani-kalpachka**
- Removed inaccurate partners.
- Added new partners.
- Rearrange and added styles to Partners page.
- Formatted svg files.

### CODE REVIEW

File: public/img/partners/irisSolutions.svg
Comment (sashko9807) [inline public/img/partners/irisSolutions.svg:1]: The partner logo here should be IRIS Solutions. Iris pay is one of the services they offer. 
Do we have another logo?

File: src/components/admin/partners/helpers/partnersData.ts [hunk 1]
```diff
@@ -115,18 +113,28 @@ export const allPartners = [
...
-    website: partnerUrls.allChannels,
+    name: 'crossroadsBulgaria',
+    image: '/img/partners/crossRoads.svg',
+    website: partnerUrls.crossroadsBulgaria,
   },
   {
-    name: 'brainDonors',
-    image: '/img/partners/brainDonors.svg',
-    website: partnerUrls.brainDonors,
+    name: 'crossroadsBulgaria',
```
Comment (sashko9807) [inline src/components/admin/partners/helpers/partnersData.ts:121]: Name should be estProduction, I believe?

**Review by sashko9807 — CHANGES_REQUESTED** [review]

File: src/components/admin/partners/helpers/partnersData.ts [hunk 2]
```diff
@@ -115,18 +113,28 @@ export const allPartners = [
...
+    image: '/img/partners/estProduction.svg',
+    website: partnerUrls.estProduction,
```
Comment (sashko9807) [inline src/components/admin/partners/helpers/partnersData.ts:123]: Name should be estProduction I believe.

File: public/img/partners/irisSolutions.svg
Comment (ani-kalpachka) [reply public/img/partners/irisSolutions.svg:1]: I see. This was the provided logo by them. We can discuss it on the meeting today.

File: src/components/admin/partners/helpers/partnersData.ts (hunk 2 above)
Comment (ani-kalpachka) [reply src/components/admin/partners/helpers/partnersData.ts:123]: Thanks for spotting it! I will fix it.

File: public/img/partners/allChannels.svg
Comment (sashko9807) [inline public/img/partners/allChannels.svg:1]: If allChannels is removed from partners, do we need to keep their logo?

File: public/img/partners/brainDonors.svg
Comment (sashko9807) [inline public/img/partners/brainDonors.svg:1]: If brainDonors has been removed from partners, do we need to keep their logo?

Comment (ani-kalpachka) [reply public/img/partners/brainDonors.svg:1]: Updated. Thanks!

**Review by sashko9807 — APPROVED** [review]

**Pull request merged**